File: backend/api/src/main/scala/com/github/ai/split/api/response/DeleteMemberResponse.scala
package com.github.ai.split.api.response

import com.github.ai.split.api.GroupDto
import zio.json.{DeriveJsonEncoder, JsonEncoder}

case class DeleteMemberResponse(
  group: GroupDto
)

object DeleteMemberResponse {
  implicit val encoder: JsonEncoder[DeleteMemberResponse] = DeriveJsonEncoder.gen[DeleteMemberResponse]
}

File: backend/api/src/main/java/com/github/ai/split/api/response/DeleteMemberResponse.java
package com.github.ai.split.api.response;

import com.github.ai.split.api.GroupDto;
public class DeleteMemberResponse {

    public GroupDto group = new GroupDto();

    public DeleteMemberResponse() {}

    public DeleteMemberResponse(GroupDto group) {
        this.group = group != null ? group : new GroupDto();
    }
}